modular_skyrat/master_files/code/modules/events/spacevine.dm

```diff
@@ -474,41 +478,41 @@
 // Pulses radiation on growth
 /datum/spacevine_mutation/radiation
 	name = "radiation pulsing"
-	hue = "#ffef62"
+	plant_color = "#ffef62"
 	severity = 3
 	quality = NEGATIVE
 
-/datum/spacevine_mutation/radiation/on_grow(obj/structure/spacevine/holder)
-	radiation_pulse(holder, 100, 3)
+/datum/spacevine_mutation/radiation/on_grow(obj/structure/spacevine/vine_object)
+	radiation_pulse(vine_object, 100, 3)
```

Comment:
Radiation was changed, I don't think this would work properly anymore, and I fear the range of that pulse (which would be a max range of 100 tiles, which I doubt is what's intended :) )